pkg/cache/cache_from.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/constants"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+)
+
+// CacheFrom is a list of images to import cache from.
+type CacheFrom []string
+
+type ImageCtrlInterface interface {
```

Comment:
I think we should add a description here since it is exported to the registry pkg

Reply:
It's no longer exported now, but it was a valid comment 🙏